joomla/joomla-cms
# Update CodeMirror to 5.60.0

### Summary of Changes

Update CodeMirror to 5.60.0 using https://github.com/okonomiyaki3000/joomla-codemirror-builder
Changes: https://github.com/codemirror/CodeMirror/compare/5.56.0...5.60.0

This also patches the [CVE-2020-7760](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2020-7760) 

### Testing Instructions

- apply the patch
- Make sure Codemirror still works

### Actual result BEFORE applying this Pull Request

Running version 5.56.0

### Expected result AFTER applying this Pull Request

Running version 5.60.0

### Documentation Changes Required

none
